draft-ietf-lamps-pq-composite-kem.md

```diff
@@ -1201,7 +1174,7 @@ Label values are provided as ASCII strings, but MUST be converted into binary st
 For example:
 
 * "\.//^\" in hexadecimal is "5c2e2f2f5e5c"
-* "QSF-MLKEM768-P256-HMACSHA256" in hexadecimal is "5153462d4d4c4b454d3736382d503235362d484d4143534841323536"
+* "QSF-MLKEM768-P256-SHA3-256" in hexadecimal is "5153462d4d4c4b454d3736382d503235362d534841332d323536"
```

Comment:
How much is this document aligning with draft-irtf-cfrg-concrete-hybrid-kems?  Because their label is "QSF-P256-MLKEM768-SHAKE256-SHA3256".  Which is not even consistent with their KEM naming, I've raised that here: https://github.com/cfrg/draft-irtf-cfrg-concrete-hybrid-kems/issues/23